gadget.yaml

```diff
@@ -45,7 +45,11 @@ volumes:
             target: EFI/boot/grubx64.efi
           - source: shim.efi.signed
             target: EFI/boot/bootx64.efi
-      # XXX: should we define the  "ubuntu-save" partition here too?
+      - name: ubuntu-save
+        role: system-save
+        filesystem: ext4
+        type: 83,0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        size: 8M
```

Comment:
I ran into some issues with formatting this device as LUKS2 and then mounting it. We may need to use a larger size.

Reply:
I don't think 8MB is enough for all the luks headers & backup, as well as ext4.

Is the intention for it to have 8MB of usable free disk space, once it's encrypted and mounted?